torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -990,7 +990,7 @@ def use_thread_local_cached_output_tensor(idx, output):
             output_buffer = V.graph.graph_outputs[idx]
             if isinstance(output_buffer, ir.BaseView):
                 output_storage = output_buffer.unwrap_view()
-                if isinstance(output_storage.data, ir.ConstantBuffer):
+                if isinstance(output_storage.data, ir.ConstantBuffer):  # type: ignore[attr-defined] # next PR
```

Comment:
worth adding a get_data method that throws to `IRNode`  and is typed appropriately